Library/Formula/cctools.rb

```diff
@@ -11,6 +11,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation true
```

Comment:
Can nix the bool argument here, I think - simply `does_not_need_relocation` is clear enough on its own.